guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,228 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 in-place by using Leapp
```

Comment:
```suggestion
= Upgrading {Project} or {SmartProxy} to {EL-abbr} 9 in-place by using Leapp
```
Can we make the title shorter somehow? At least by defining an abbreviation? I'm thinking:
- in `attributes-base.adoc`, define `:EL-abbr: {EL}` (include after `:EL:`)
- in `attributes-satellite.adoc`, define `:EL-abbr: RHEL`